### PR TITLE
Lalit

### PR DESCRIPTION
**Pull request by lalitsaud1997**

### CODE REVIEW

File: 04_JS_statements.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+<body>
+<div class="js_Statements">
```
Comment (lalitsaud1997) [inline 04_JS_statements.html:4]: js statement division

**Review by lalitsaud1997 — COMMENTED** [review]
nice

**Review by saudG1997 — APPROVED** [review]
JS statements div

**Review by lalitsaud1997 — COMMENTED** [review]
lalit

**Pull request merged**